gateway/README.md

```diff
@@ -961,12 +962,11 @@ data that was late or in the future respectively.
       ]
     }
 
-An alternative to using the above `StatsDelay` is to have the gateway provide a
-signalfx datapoint endpoint that contains the same information but can be
-scraped by our Smart Agent.  If you set the config
-`InternalMetricsListenerAddress` to a host port combination and configure
-the smart gateway to scrape that for internal metrics at `/internal-metrics`
-it would accomplish the same thing.
+If you set the config `InternalMetricsListenerAddress` to a host port combination
```

Comment:
is: If you set the config InternalMetricsListenerAddress
change to: You can set the InternalMetricsListenerAddress configuration parameter

is: you will need to also sert EmitDebugMetrics to true.
change to: you will need to also set EmitDebugMetrics to true.